bin/firefox-schema-import

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+// vim: set ft=javascript:
```

Comment:
Hmm, shouldn't vim recognize this via the shebang already? I'm -1 settings like that tbh.